packages/platform-sdk-ada/src/services/client.ts

```diff
@@ -97,10 +72,11 @@ export class ClientService implements Contracts.ClientService {
 	}
 
 	public async wallet(id: string): Promise<Contracts.WalletData> {
-		const { usedSpendAddresses, usedChangeAddresses } = await this.usedAddressesForAccount(id);
+		const { usedSpendAddresses, usedChangeAddresses } = await usedAddressesForAccount(this.#config, id);
```

Comment:
Think we should swap the argument here so it matches `fetchTransaction(id, this.#config)`.